File: Array-2/countEvens/CountEvens.java
package countEvens;

public class CountEvens {
    //Retorne a quantidade de números paraes dentro do Array dado ("nums").

    public int countEvens(int[] nums) {
        int cont = 0;
        for (int n :
                nums) {
            if (n%2 == 0) cont++;
        }
        return cont;
    }
}

File: Python/List-2/count_evens.py
"""
Dada uma lista numérica "nums", retorne a quantidade de números pares que ela
possui.
"""


def count_evens(nums):
    cont = 0
    for n in nums:
        if n % 2 == 0:
            cont += 1
    return cont


print(count_evens([1, 2, 3, 4]))
